csrc/fusion.h

```diff
@@ -189,8 +189,11 @@ class Fusion : public IrContainer {
   bankConflictInfo(const CompileParams& compile_params = CompileParams());
 
   //! Return a list of topologically sorted expressions. This only includes
-  //! exprs required to genereate registered outputs.
+  //! exprs required to generate registered outputs.
   std::vector<Expr*> exprs();
```

Comment:
Do we still need this non-const version?

Reply:
hrm, good question! i'll look into that.

Reply:
compiles fine locally w/o it. will add you to https://github.com/NVIDIA/Fuser/pull/1798 once it passes CI.